tests/conftest.py

```diff
@@ -303,26 +303,41 @@ def _slurm_not_installed():
 def _subprocess_setup(request, cfg=None, uwsgi=False):
     """setup the supervisor"""
     import os
+    import sirepo.job
+    import sirepo.pkcli.service
     from pykern.pkcollections import PKDict
 
+    ports = PKDict()
+
+    # different port than default so can run tests when supervisor running
+    def _next_conftest_port(port_key, port_pkg):
```

Comment:
`_port()`. We are in conftest so we don't need to add this name to a function. This is nested function so the space/use is limited. I do like "port_key" though. :)